bucket/hcloud.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.21.0",
+    "description": "hcloud: Command-line interface for Hetzner Cloud",
+    "homepage": "https://github.com/hetznercloud/cli",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/hetznercloud/cli/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hetznercloud/cli/releases/download/v1.21.0/hcloud-windows-amd64.zip",
+            "hash": "7478f92d977d1151c93f3cbb13ecb0b1f8f0f4cff88310e8f8cc1acfcbf023dd"
+        },
+        "32bit": {
+            "url": "https://github.com/hetznercloud/cli/releases/download/v1.21.0/hcloud-windows-386.zip",
+            "hash": "b990eca66bbc41a0086ac4e79b69b6777dc57b562e78faa84297624c503893c5"
+        }
+    },
+    "bin": "hcloud.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/hetznercloud/cli/releases/download/v$version/hcloud-windows-amd64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/hetznercloud/cli/releases/download/v$version/hcloud-windows-386.zip"
+            }
+        }
+    }

```

Comment:
Autoupdate hash url

```suggestion
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
    }
```

Reply:
Thank you for the review and your suggestions. I've included them in the PR now and tested it, too.